source/tutorials/multi-boot/dual-boot-linux.rst

```diff
@@ -1,14 +1,16 @@
 .. _dual-boot-linux:
 
-Dual-boot |CL-ATTR| with Another GRUB-based Linux\* Distro
+Dual-boot |CL-ATTR| with Any GRUB-based Linux\* Distro
```

Comment:
It's not clear to me why you're including `dual-boot-linux` and `multi-boot` on this PR while you're already addressing them in GH-909. Was this done by mistake? If so, simply remove the latter two docs.  
Then we'll use `enable-systemd-boot-menu` as the only doc on this PR. If otherwise, please advise.